models/image.py

```diff
@@ -1720,6 +1758,65 @@ def score(self):
     def score(self, value):
         self._score = value
 
+    @property
+    def psfflux(self):
+        """
+        The flux of the PSF (2D float array).
```

Comment:
It would be worth documenting more exactly what this is supposed to be.

I'm assuming it is : this must be an odd-sized 2d array where the PSF is centered on the center pixel.  The entire array must sum to 1.  It represents the PSF at the same resolution as the image resolution, evaluated at the center of the image.  

Storing the PSF at image resolution isn't enough to really reconstruct the PSF for sub-pixel shifts, unless you're sufficiently oversampled (which we usually won't be in LS4).  psfex actually stores the PSF at a different resolution so it can do sub-pixel shifts without spreading the PSF out.

Becuase of this, and because this doesn't capture a variable psf, the property psfflux isn't really the right thing to use for everything you'd want to use a psf for.  It's worth putting in the documentation what it *is* good for.